internal/indexer/definitions/btn.yaml

```diff
@@ -102,4 +102,5 @@ irc:
           - torrentId
 
     match:
+      infourl: "/torrents.php?id={{ .torrentId }}"
```

Comment:
We capture the ID of the torrent itself here...
But `/torrents.php?id={{ .torrentId }}` takes you to the torrent group which would be the respective episode for example..
Correct would be https://broadcasthe.net/torrents.php?id=000000&torrentid=0000000

Hence this resulting in 404 aswell.